diff --git a/Marlin/pins_TRIGORILLA_13.h b/Marlin/pins_TRIGORILLA_13.h
index 0907eaf28..4074a5563 100644
--- a/Marlin/pins_TRIGORILLA_13.h
+++ b/Marlin/pins_TRIGORILLA_13.h
@@ -29,9 +29,14 @@
 #endif
 
 #define IS_RAMPS_EFB
-#define RAMPS_D9_PIN         44
-#define FAN2_PIN              9
-#define ORIG_E0_AUTO_FAN_PIN  9
+
+// FAN0 / D9  - Typically used for the part fan on Anycubic Delta devices
+#define FAN_PIN 9
+
+// FAN1 / D7  - Typically unused, can be allocated as Case Fan
+
+// FAN2 / D44 - Typical Extruder Fan on Anycubic Delta devices
+#define ORIG_E0_AUTO_FAN_PIN  44
 
 #include "pins_RAMPS_13.h"
 
@@ -39,3 +44,36 @@
 #undef E1_DIR_PIN
 #undef E1_ENABLE_PIN
 #undef E1_CS_PIN
+
+
+// TODO check this
+
+//////////////////////////
+// LCDs and Controllers //
+//////////////////////////
+
+#if ENABLED(ULTRA_LCD)
+
+  //
+  // LCD Display output pins
+  //
+  #if ENABLED(NEWPANEL) && ENABLED(PANEL_ONE)
+    #define LCD_PINS_D6 57
+  #endif
+
+  //
+  // LCD Display input pins
+  //
+  #if ENABLED(NEWPANEL)
+    #if ENABLED(VIKI2) || ENABLED(miniVIKI)
+      #define DOGLCD_A0        23
+    #elif ENABLED(ELB_FULL_GRAPHIC_CONTROLLER)
+	  #define BEEPER_PIN 33
+      #define LCD_BACKLIGHT_PIN 67
+    #endif
+  #elif ENABLED(MINIPANEL)
+    #define BEEPER_PIN 33
+    #define DOGLCD_A0  42
+  #endif
+
+#endif // ULTRA_LCD
